build.sbt

```diff
@@ -13,7 +13,7 @@ ThisBuild / libraryDependencySchemes ++= Seq(
   "com.github.plokhotnyuk.jsoniter-scala" % "jsoniter-scala-core" % "early-semver"
 )
 
-ThisBuild / versionPolicyIntention := Compatibility.BinaryCompatible
+ThisBuild / versionPolicyIntention := Compatibility.None
```

Comment:
This is a sign that after merging the PR the major version number needs to be increased. It will be set back to the stringent one after the release is cut.